pkgs/applications/misc/msglint/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "msglint-1.04";
+
+  setSourceRoot = "sourceRoot=`pwd`";
```

This is a very confusing way to set the `sourceRoot` to the current directory.

```suggestion
  sourceRoot = ".";
```